seed_db.sh

```diff
@@ -0,0 +1,217 @@
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 3, 'After two years we felt that we could approach your sixteen verses of the fourth chapter of Genesis. ');"
+
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 4, 'My old gentlemen felt that these words were very important too—’Thou shalt’ and ‘Do thou.’ 
+            And this was the gold from our mining: ‘Thou mayest.’ ‘Thou mayest rule over sin.’ ');"
+
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 5, 'The old gentlemen smiled and nodded and felt the years were well spent. It brought them out of their
+            Chinese shells too, and right now they are studying Greek.” ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 6, 'Samuel said, “It’s a fantastic story. And I’ve tried to follow and maybe I’ve missed somewhere. 
+            Why is this word so important?”');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 7, 'Lee’s hand shook as he filled the delicate cups. 
+            He drank his down in one gulp. “Don’t you see?” he cried. ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 8, '“The American Standard translation orders men to triumph over sin, and you can call sin ignorance. ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 9, 'The King James translation makes a promise in ‘Thou shalt,’ 
+            meaning that men will surely triumph over sin. ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_contents (story_id, line_index, content) VALUES ('$STORY_ID', 10, 'But the Hebrew word, the word timshel—‘Thou mayest’—that gives a choice. 
+            It might be the most important word in the world. 
+            That says the way is open. That throws it right back on a man.
+            For if ‘Thou mayest’—it is also true that ‘Thou mayest not.’ Don’t you see?”');"
+    done 
+
+    
+    # Story translation content 
+    docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "ALTER SEQUENCE story_translation_contents_id_seq RESTART WITH 1;"
+
+
+    for STORY_TRANSLATION_ID in 2 3 5 7 11 13 
```

Comment:
For every story_translation that exists, the respective story_translation_content lines must also exist (see [create_story_translation](https://github.com/uwblueprint/planet-read/blob/main/backend/python/app/services/implementations/story_service.py#L85) service). i.e. if you create a story translation of a 6 line story, the there must also be 6 lines of story translation content created with  empty strings. 

A quick fix is to run this forloop from 1 to 13. To diversify our data, you can make one story translation full of emtpy strings, and one that has some empty and some full lines.